bin/rspec

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
```

Comment:
This is a binstub generated by bundler. I tend to use binstubs, e.g. `bin/rspec`. Happy to remove it though. It's a bit funky in gems that include executables.